app/gui2/src/assets/base.css

```
@@ -63,8 +63,7 @@ body {
   background: #e4d4be;
   color: var(--color-text);
   transition: color 0.5s, background-color 0.5s;
-  font-family: Inter, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu,
-    Cantarell, 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
+  font-family: 'M PLUS 1', sans-serif;
   font-size: 11.5px;
   font-weight: 500;
   line-height: 174.5%;
```

```suggestion
  line-height: 20px;
```

the Figma design doesn't use exactly 20px, but i think that's the intention as some are slightly less than 20px, and some are slightly more than 20px.